src/Logging/Logger.php

```diff
@@ -94,6 +100,19 @@ public static function critical(string $sMessage, LogContext $logContext, array
         self::logContext(self::CRITICAL, $sMessage, $logContext, $aMetadata);
     }
 
+    /**
+     * Critical conditions exception
+     */
+    public static function criticalException(string $sMessage, LogContext $logContext, \Throwable $e, array $aMetadata = []): void
```

Comment:
IMHO. If this method is Your _respect_ to PSR-3 - I can understand, but otherwise, It is hard to understand the sense or the context of it.
Should the logger use exception instances to understand what is critical and what is not (to avoid agreements between developers)?

Reply:
I would say it's quite clear, when You want to log exception which is critical then use this method. Logger can not use exception instances to understand what is critical and what is not, because the same type of Exception can be critical in one context but not critical in another one.